website/content/docs/platform/k8s/vso/sources/vault/index.mdx

```diff
@@ -12,6 +12,8 @@ The Vault Secrets Operator (VSO) supports Vault as a secret source, which
 lets you seamlessly integrate VSO with a Vault instance running on any
 platform.
 
+The Vault Secrets Operator (VSO) is fully supported component of Hashicorp Vault.
```

Comment:
"is fully" ---> "is a fully"

Reply:
done here: [58f4966](https://github.com/hashicorp/vault/pull/30173/commits/58f49665b1c4206823657fadcb2de5bb62b9d81f)